blocks/amp-story/block-selector.js

```diff
@@ -48,6 +48,14 @@ class BlockSelector extends Component {
 			let className = 'component-editor__selector template-' + template;
 			if ( isBlockSelected( block.clientId ) || hasSelectedInnerBlock( block.clientId ) ) {
 				className += ' is-selected';
+
+				// Do not blur the CTA layer so the user can edit the elements within it.
+				if ( 'amp/amp-story-cta-layer' === block.name ) {
+					const ampStoryDiv = document.getElementById( 'block-' + block.clientId );
+					if ( ampStoryDiv ) {
+						ampStoryDiv.classList.add( 'is-selected' );
```

Comment:
@jacobschweitzer What seems to be the issue here with blurring? Perhaps that's something that @mehigh can take a look at within CSS instead? CTA Layer shouldn't probably be blurred at any point since it's always the top layer -- so it wouldn't be possible to select a layer that's on top of CTA Layer.

Mike, do you think you could look into CTA Layer to make sure that it's not being blurred when edited? There seems to be another new CSS-related issue with CTA Layer -- The CTA Layer should always take 20% of the Page height, however, it looks like the CTA Layer has the same height as the other layers:

![screen shot 2018-11-22 at 9 25 59 am](https://user-images.githubusercontent.com/3294597/48902869-aba38600-ee38-11e8-9149-503149728527.png)

Reply:
Also note that the Layer Cards ( block-selector.js ) will be removed within #1630: that's something that will not exist soon and will be replaced by the Block Navigator on the left.

Reply:
@miina Here is what I was seeing with the blurring issue:
<a href="https://cl.ly/e1384d9be219" target="_blank"><img src="https://duaw26jehqd4r.cloudfront.net/items/2J1K0K3w2r43303b121V/Screen%20Recording%202018-11-22%20at%2003.34%20PM.gif" style="display: block;height: auto;width: 100%;"/></a>

Reply:
The underlying issue seems to be that the child block gets the `is-selected` class so it is not blurry, but its parent (`div.amp-story-cta-layer`) does not have `is-selected` so it is blurred which makes all its child elements blurred.

Reply:
@jacobschweitzer In the `helpers.js` file there is the method `maybeIsSelectedParentClass` which is used for adding `is-selected-parent` class to page and also layers -- except for CTA Layer -- perhaps you could implement this for the CTA Layer as well? This might fix the issue and if it does not right away, it would give a CSS selector for fixing the issue via CSS, as it's done for the other layers at this moment. Thoughts?

Reply:
Ok thanks @miina that looks like a good approach, I'll start working on that.

Reply:
For reference: https://github.com/Automattic/amp-wp/blob/amp-stories/assets/js/amp-story-editor-blocks.js#L228-L230